pages/_app.js

```diff
@@ -1,18 +1,38 @@
 import App from 'next/app';
 import GlobalStyle from '@catho/quantum/GlobalStyle';
+import { Hide } from '@catho/quantum';
 
 export default class HomePage extends App {
   componentDidMount() {
     const hiddenMenu = document.getElementById('hiddenMenu');
     document.getElementById('menuButton').addEventListener('click', showMenu);
+    const mobileMenu = document.getElementById('MobileMenu');
     function showMenu() {
-      const { classList } = hiddenMenu;
-      if (classList.contains('HideMenu')) {
-        classList.add('ShowMenu');
-        classList.remove('HideMenu');
+      // const hiddenMenuclasses = hiddenMenu.classList;
+      // const mobileMenuclasses = mobileMenu.classList;
+      if (
+        hiddenMenu.classList.contains('HideMenu') &&
+        window.innerWidth > 1023
```

Comment:
This condition can be placed in a variable with a name that represents the expected result, for example: `const showHiddenMenu = window.innerWidth > 1023;`

```suggestion
      const showHiddenMenu = window.innerWidth > 1023;
      if (
        hiddenMenu.classList.contains('HideMenu') &&
        showHiddenMenu
```